doc/getting-started.md

```diff
@@ -35,6 +41,20 @@ This article is a guide for installing the base Uno.Toolkit library, additional
         </ResourceDictionary.MergedDictionaries>
     </ResourceDictionary>
     ```
+### [**dotnet new**](#tab/dotnet-new)
```

Comment:
you shouldn't use numbered items here
it is not step1 followed by step2, but rather option1 vs option2

Reply:
and, for "2." you need to included the part that adds the \<ToolkitResources> (step.4 from the Wizard section)